src/Whitelist/VulnerabilityExcluder.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { existsSync, readFileSync } from "fs";
+import path from 'path';
+
+import { OssIndexServerResult } from "../Types/OssIndexServerResult";
+
+const whitelistFilePathPwd = path.join(process.cwd(), 'auditjs.json');
+
+export abstract class VulnerabilityExcluder {
+  public static filterVulnerabilities(results: Array<OssIndexServerResult>, whitelistFilePath: string = whitelistFilePathPwd): Array<OssIndexServerResult> {
+    if (existsSync(whitelistFilePath)) {
+      const json = readFileSync(whitelistFilePath);
```

Comment:
trivial: from my comment on line 16, i believe it would look something like `readFileSync(whitelistFilePath, { flag: 'r+' })`

Reply:
Got a triple double in: fd46cb8

Reply:
🤣